src/navigation/menu.es6

```diff
@@ -59,8 +55,8 @@ export const sortMenu = (markets) => {
    //Sort market
    if($.isArray(markets)) {
 
-      const rank = { "forex": 1, "indices": 2, "otc stocks": 3, "commodities": 4, "volatility indices": 5 };
-      markets = _.sortBy( markets, (o) => rank[o.display_name.toLowerCase()]);
+      const rank = { "forex": 1, "indices": 2, "otc stocks": 3, "commodities": 4, "volidx": 5 };
+      markets = _.sortBy( markets, (o) => rank[o.name.toLowerCase()]);
```

Comment:
Can't we use our marketUtils in common for this?

Reply:
yes im planning to do so as well 👍 , but if need quick fix, then this is the fastest solution